packages/fuel-indexer-macros/src/schema.rs

```diff
@@ -91,13 +109,23 @@ fn process_fk_field<'a>(
     let typ = process_type(types, &field_type, false);
     let ident = format_ident! {"{}", field_name.to_lowercase()};
 
+    let (_, column_type) = get_column_type(typ.clone());
+
+    // TODO: We should make a formal decision as to whether we'll allow
```

Comment:
Since both Postgres and SQLite support `NULL` foreign keys, we should do the same (again, for everything except `ID` types). Apologies for not including this in the original issue.